railties/lib/rails/generators/actions.rb

```diff
@@ -344,6 +343,13 @@ def with_indentation(&block) # :doc:
         ensure
           @indentation -= 1
         end
+
+        # Append string to the +Gemfile+ with a newline if necessary
```

Comment:
I think this method is not limited to `+Gemfile+`, how is it?

Reply:
While I understand that this whole file is kind of dedicated to generating Gemfile and the method is private, you are right. This method is not technically limited for Gemfile.

Thank you for the heads up! Just fixed it.

Reply:
While I understand that this whole file is kind of dedicated to generating Gemfile and the method is private, you are right. This method is not technically limited for Gemfile.

Thank you for the heads up @y-yagi ! Just fixed it.